Userland/Libraries/LibWeb/CSS/StyleProperties.cpp

```diff
@@ -355,6 +355,22 @@ Optional<CSS::TextAlign> StyleProperties::text_align() const
     }
 }
 
+Optional<CSS::PointerEvents> StyleProperties::pointer_events() const
+{
+    auto value = property(CSS::PropertyID::PointerEvents);
+    if (!value.has_value() || !value.value()->is_identifier())
+        return {};
+
+    switch (static_cast<const IdentifierStyleValue&>(*value.value()).id()) {
```

Comment:
There's no need to do the cast here. I changed how StyleValues work a bit since you started on this, but you can just do:
```c++
switch (value.value()->to_identifier()) {
```